### ISSUE TITLE
Postman / Newman Spec

### ISSUE THREAD

**Issue by tisto** [plone/plone.restapi#272]
Postman allows creating "collections" which is basically a group of individual HTTP calls. Newman allows executing a Postman collection via command line. 

We could investigate if this would allow us to create a plone.restapi "spec" that we could use as an acceptance test for plone.restapi. This could also help us to keep plone.restapi and Guiolltina API in sync.

@prakharjoshi you offered to look into this some time ago. Are you still interested in working on that?

cc @bloodbare @vangheem @sneridagh

**Comment by prakharjoshi** [comment]
@tisto Yes I am still willing to work on that. I will start of by creating collection first and then lets see how newman behave with that collection.

**Comment by tisto** [comment]
@prakharjoshi any news on this? Do you still plan to work on this?

**Comment by MAX-786** [comment]
Any update on this @prakharjoshi @tisto ? , I would love to help in investigate this! Thanks

**Comment by jensens** [comment]
Closing as outdated (8 years old). Postman/Newman spec from 2017. plone.restapi has comprehensive test coverage and docs at docs.plone.org now.
If you think this issue is still relevant please reopen it. Thank you for the original report!
